src/components/admin/modal-wrappers/generic-modal-wrapper/GenericModalWrapper.tsx

```diff
@@ -103,26 +89,16 @@ export const GenericModalWrapper = <TFormValues, TFormRef>({
                 </Modal.Actions>
             </Modal>
 
-            <ConfirmationModal
-                isOpen={showFormConfirmModal}
-                isButtonsDisabled={isSubmitting}
-                title={formConfirmTitle}
-                onConfirm={onConfirmAction}
-                onCancel={handleCancel}
+            <ConfirmationModals
```

Comment:
I believe you can use spread operator here, check it and confirm please
```suggestion
            <ConfirmationModals
                isSubmitting={isSubmitting}
                onClose={onClose}
                {...confirmationProps} 
            />
```